### DISCUSSION TITLE
Not sure if an issue, so I start a discussion first off

### DISCUSSION THREAD

**Discussion by Sordoodledom** [FrozenBlock/WilderWild#170] [General]
Hello Dev Team, 

first of all I want to thank you for your awesome mod. It gave the Minecraft world a very fun and unique touch which I don't want to loose again. Great work so far. 
Now to my question: 

I've done some exploring recently and tried to find a donkey in the plain biomes. Unfortunately I was not lucky finding one in more than 20 different plain biomes (regular plains, snowy_plains, sunflower_plains). Could there be an issue with the entity generation? I've looked up your code, there are several files which are used to generate the biomes. Most of them seems to use the Minecraft default properties. But I'm by no means a mod developer and I'm not sure at all if I was searching correctly. If I can provide you with any data please let me know.  I'll try to help where is can. 

Thank you in advance. 
Kind regards
Sordoodledom

**Comment by Soro335** [comment]
We didn't change vanilla mob spawn

**Reply by Sordoodledom** [reply]
In this case I have to extend the exploration once again. Thank you for the quick reply.
